test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -45,5 +45,18 @@ public static string GetConfigContent(ServerType serverType, string iisConfig, s
 
             return content;
         }
+
+        public static string GetNginxConfigContent(ServerType serverType,  string nginxConfig)
```

Comment:
Ugh, ugly. Can we put it into hosting too?

Reply:
I'm planning on it, I was trying to keep this constrained to one deployer at a time.

Reply:
Why can't you drop the original GetConfigContent.

Reply:
GetConfigContent is still used in the compression tests.